src/worker/testplane-facade.ts

```diff
@@ -31,6 +31,10 @@ module.exports = class TestplaneFacade {
         return this.promise;
     }
 
+    cancel(): void {
+        RuntimeConfig.getInstance().replServer?.close();
```

Comment:
If `replServer` doesn't exist (user do not use `--repl` option) then nothing will not fail here.